src/nethsec/mwan/__init__.py

```diff
@@ -420,6 +424,8 @@ def index_rules(e_uci: EUci) -> list[dict]:
             rule_data['destination_address'] = rule_value['dest_ip']
         if 'dest_port' in rule_value:
             rule_data['destination_port'] = rule_value['dest_port']
+        if 'sticky' in rule_value:
+            rule_data['sticky'] = rule_value['sticky'] == '1'
```

Comment:
Same as above

Reply:
hum just tested, it does not work
```
{
  "values": [
    {
      "name": "ns_aa",
      "policy": {
        "name": "ns_default",
        "label": "Default"
      },
      "label": "aa",
      "protocol": "all",
      "sticky": "1"
    },
    {
      "name": "ns_default_rule",
      "policy": {
        "name": "ns_default",
        "label": "Default"
      },
      "label": "Default Rule",
      "protocol": "all",
      "sticky": "1"
    }
  ]
}
```

Reply:
look at the ouput of ` index_rules` with your suggestion

```
root@NethSec:~# echo '{}' | /usr/libexec/rpcd/ns.mwan call index_rules | jq
{
  "values": [
    {
      "name": "ns_aa",
      "policy": {
        "name": "ns_default",
        "label": "Default"
      },
      "label": "aa",
      "protocol": "all",
      "sticky": "1"
    },
    {
      "name": "ns_default_rule",
      "policy": {
        "name": "ns_default",
        "label": "Default"
      },
      "label": "Default Rule",
      "protocol": "all",
      "sticky": "1"
    }
  ]
}
```

look at with my code

```
root@NethSec:~# echo '{}' | /usr/libexec/rpcd/ns.mwan call index_rules | jq
{
  "values": [
    {
      "name": "ns_aa",
      "policy": {
        "name": "ns_default",
        "label": "Default"
      },
      "label": "aa",
      "protocol": "all",
      "sticky": true
    },
    {
      "name": "ns_default_rule",
      "policy": {
        "name": "ns_default",
        "label": "Default"
      },
      "label": "Default Rule",
      "protocol": "all",
      "sticky": true
    }
  ]
}
```

the UI now expect true/false like requesting by you

Reply:
Okay, I've got the issue, we're using `utils.get_all_by_type(e_uci, 'mwan3', 'rule')` which since it's a dictionary we're not casting correctly the variable types with EUCI, then it's good :ok_hand: